src/components/info/filtersSummary.js

```diff
@@ -86,7 +87,19 @@ class FiltersSummary extends React.Component {
       .sort((a, b) => a.value < b.value ? -1 : a.value > b.value ? 1 : 0)
       .map((item) => {
         let label = `${item.value}`;
-        if (filterName!==strainSymbol) label+= ` (${this.props.totalStateCounts[filterName].get(item.value)})`;
+        if (filterName!==strainSymbol) {
+          /* Add the _total_ occurrences in parentheses. We don't compute the intersections / visible values here -
+          e.g. we can have "England (5)" "North America (100)" even though such an intersection will deselect everything.
+          If we have two trees shown we show both values.
+          */
+          const tree1count = this.props.totalStateCounts[filterName]?.get(item.value) ?? 0;
+          if (this.props.totalStateCountsSecondTree) {
+            const tree2count = this.props.totalStateCountsSecondTree[filterName]?.get(item.value) ?? 0;
+            label+=` (L: ${tree1count}, R: ${tree2count})`;
```

Comment:
Here's an example of how this looks:

<img width="307" alt="image" src="https://github.com/nextstrain/auspice/assets/8350992/41d8beeb-a880-48e3-8806-b4957e283533">